fe/fe-core/src/main/java/com/starrocks/http/action/SessionAction.java

```diff
@@ -44,6 +44,7 @@ public class SessionAction extends WebBaseAction {
         SESSION_TABLE_HEADER.add("Cluster");
         SESSION_TABLE_HEADER.add("Db");
         SESSION_TABLE_HEADER.add("Command");
+        SESSION_TABLE_HEADER.add("ConnectionStartTime");
```

Comment:
Do we need `CommandStarTime`?

Reply:
it's introduced in this PR. https://github.com/StarRocks/starrocks/pull/5590

But to be consistent, we have to show it on http dashboard too, like here.

<img width="1547" alt="image" src="https://user-images.githubusercontent.com/1081215/184625099-2ebbfb65-8caa-4a79-9051-1006e12fef08.png">